docs/security/trust/content_trust.md

```diff
@@ -0,0 +1,262 @@
+<!--[metadata]>
++++
+title = "Content trust in Docker"
+description = "Enabling content trust in Docker"
+keywords = ["trust, security, docker,  documentation"]
+[menu.main]
+parent= "smn_content_trust"
+weight=-1
++++
+<![end-metadata]-->
+
+# Content trust in Docker
+
+When transferring data among networked systems, *trust* is a central concern.
+When using Docker you push and pull images (data) to a registry. Currently,
+trust is supported for users of Docker's public Hub. It is not available for
+Docker Trusted Registry or for private registries. 
+
+## Understand trust in Docker 
+
+Content trust allows operations with a remote Docker registry to enforce
```

Comment:
I find the `operations with a remote Docker registry` confusing.

What about: `Content trust provides the ability to use digital signatures for data sent to and received from remote Docker registries. These signatures allow client-side verification of the integrity and publisher of specific image tags.`